### DISCUSSION TITLE
Mulighet for å se avviksbeskrivelse i Tavla

### DISCUSSION THREAD

**Discussion by wfeiring** [entur/tavla#316] [Ideas]
Per i dag får man opp et ikon om det er avvik på linjene på Tavla, men det er ofte mye ubrukt plass der man kunne vist beskrivelsen også. Evt. hvis det blir for langt kunne man kanskje hatt det på to linjer?

**Comment by akselaase** [comment]
Mulig noe kan merges fra AtB - https://github.com/AtB-AS/tavla/pull/30?

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
